### PR TITLE
Add note about grpmax

### PR DESCRIPTION
**Pull request by ch-kr**
Added note that grpmax does not contain filtering allele frequency. Looking at this again, should we just remove grpmax text from the faf md?

### CODE REVIEW

File: browser/help/topics/faf.md [hunk 1]
```diff
@@ -9,10 +9,4 @@ Note that this annotation contains filtering allele frequency information from t
...
-
-### <a name="grpmax"></a>Group maximum allele frequency
-
-This annotation (abbreviated "`grpmax`") contains allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked genetic ancestry groups with the highest **AF**.
-
-For gnomAD v4, this calculation excludes Amish (`ami`), Ashkenazi Jewish (`asj`), European Finnish (`fin`), Middle Eastern (`mid`), and "Remaining Individuals" (`rmi`) groups. For gnomAD v2, this calculation excludes Ashkenazi Jewish (`asj`), European Finnish (`fin`), and "Remaining Individuals" (`rmi`) groups.
+On the browser, this annotation is directly available on the variant page. In the VCF and Hail Tables, this annotation (abbreviated "`faf`") is computed globally and for each genetic ancestry group. Filtering allele frequencies (FAFs) for each genetic ancestry group specific are listed separately with 95% and 99% CIs.
```
Comment (klaricch) [inline browser/help/topics/faf.md:12]:
```suggestion
On the browser, this annotation is directly available on the variant page. In the VCF and Hail Tables, this annotation (abbreviated "`faf`") is computed globally and for each genetic ancestry group. Filtering allele frequencies (FAFs) for each genetic ancestry group are listed separately with 95% and 99% CIs.
```

File: browser/help/topics/faf.md [hunk 2]
```diff
@@ -9,10 +9,4 @@ Note that this annotation contains filtering allele frequency information from t
```
Comment (klaricch) [inline browser/help/topics/faf.md:9]: "Note that this annotation contains filtering allele frequency information from the genetic ancestry group with the highest **FAF**" 

We should clarify that this part is referring to the grpmax faf (or whatever we settled on for a name) and not just faf

File: browser/help/topics/faf.md (hunk 1 above)
Comment (klaricch) [inline browser/help/topics/faf.md:12]: should clarify that grpmax faf is on the brower, but all  faf is in the Tables

or maybe change the id to grpmax faf and edit some of the text?

**Review by klaricch — CHANGES_REQUESTED** [review]

File: browser/help/topics/faf.md (hunk 2 above)
Comment (ch-kr) [reply browser/help/topics/faf.md:9]: will add a sentence -- let me know what you think

**Review by klaricch — CHANGES_REQUESTED** [review]
just one small thing

**Review by klaricch — APPROVED** [review]

**Pull request merged**